ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/JavaCodeGenerator.java

```diff
@@ -75,6 +80,37 @@ public void generate(
     private void generateCommands() throws IOException, TemplateException {
         for (final Ds3Request request : spec.getRequests()) {
             generateRequest(request);
+            generateResponse(request);
+        }
+    }
+
+    private void generateResponse(final Ds3Request ds3Request) throws IOException, TemplateException {
+        final Template tmpl = getResponseTemplate(ds3Request);
+
+        final Response response = getResponse(ds3Request);
+        final Path responsePath = getPath(ds3Request, response.getName());
+
+        LOG.info("Getting outputstream for file:" + responsePath.toString());
+
+        try (final OutputStream outStream = fileUtils.getOutputFile(responsePath);
+             final Writer writer = new OutputStreamWriter(outStream)) {
+            tmpl.process(response, writer);
```

Comment:
The line spacing isn't off: there are two lines within the try(...) section, which is causing the different indentations because they are in different blocks.